posts/istio-architectures-generated-by-jhipster.md

```diff
@@ -0,0 +1,168 @@
+---
+authors:
+- Fabien Arrault
+tags:
+- Kubernetes
+- Istio
+- JHipster
+- Architecture
+- Cloud
+- Microservices
+date: 2018-12-07T12:00:00.000Z
+title: "Istio architectures generated by JHipster"
+image: https://blog.ippon.fr/content/images/2018/11/Istio.jpg  
+---
+
+[JHipster](https://www.jhipster.tech/) is able to generate Kubernetes deployment descriptors for the applications it generates. Optionally, he recently proposes to add Istio specific configuration in these descriptors.
+
+After a brief introduction to Istio, the currently most known ServiceMesh, we will see that the microservice architectures associated with these deployments can be very different depending on the generation options chosen ; and study their associated characteristics.
```

Comment:
...currently the most well known Service Mesh...
...chosen and... (erroneous semicolon)